boards/qemu-i386/include/board.h

```diff
@@ -18,8 +18,8 @@
  * @author      René Kijewski <rene.kijewski@fu-berlin.de>
  */
 
-#ifndef __RIOT__BOARDS__QEMU_I386__BOARD__H
-#define __RIOT__BOARDS__QEMU_I386__BOARD__H
+#ifndef RIOT__BOARDS__QEMU_I386__BOARD__H_
+#define RIOT__BOARDS__QEMU_I386__BOARD__H_
```

Comment:
rename this, double underscores are not allowed (also it does not follow the pattern of the rest of the headers)